### ISSUE TITLE
How to do inference on a batch of images

### ISSUE THREAD

**Issue by Hegelim** [PaddlePaddle/PaddleOCR#9957]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

## Environment
- Ubuntu 18.04
- Paddle v2.5
- Python 3.7

## Issue
Let's say I have just fine-tuned a pretrained model based on ch_PPOCRv3_rec_distillation,and I want to do some benchmark tests, for example, test the speed, accuracy, etc. How could I do inference based on a folder of images? I read #3854 and some other posts but could not find any good answers. Currently, I experimented 2 ways and neither of them works:
1. call `paddleOCR()` and then `ocr.ocr()` in the code just like shown [here](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/quickstart_en.md). However, there does not seem to an argument which allows me to pass in a specific weight/pretrained model
2. use the inference code directly as [here](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/inference_ppocr_en.md). However, problem of this one is seems I cannot pass in a folder of images

Any suggestions would be appreciated.

**Comment by thanhsang298** [comment]
I use `en_PP-OCRv3_rec.yml ` config and run this for my inference phase:
```
python tools/infer_rec.py -c configs/rec/PP-OCRv3/en_PP-OCRv3_rec.yml \
                    -o Global.checkpoints="output/v3_en_mobile/best_accuracy" \
                    -o Global.infer_img="./train_data/rec/val"
``` 
in the config file:
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/51918703/e3e1ff88-d439-42f2-864b-b9d6b0d10a9d)

**Comment by andyjpaddle** [comment]
use the inference code directly as [here](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/inference_ppocr_en.md). and set `image_dir` to your batch image directory.  Then set `warmup=True` and `benchmark=True`
